layouts/shortcodes/gallery.html

```diff
@@ -2,11 +2,12 @@
 <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
 <script src="https://cdnjs.cloudflare.com/ajax/libs/lightbox2/2.11.1/js/lightbox.min.js" integrity="sha256-CtKylYan+AJuoH8jrMht1+1PMhMqrKnB8K5g012WN5I=" crossorigin="anonymous"></script>
 <style>
-li.image-gallery {float: left; display: block; margin: 0 0 5% 5%; width: 20%;}
+
+li.image-gallery {float: left; display: block; margin: 2% 2% 2% 2%; width: 20%;}
```

Comment:
those changes come from the centering, a separate pull request you did not yet merge. maybe merge the other one first?